spec/service/cloud_watch_service_spec.rb

```diff
@@ -0,0 +1,71 @@
+require "rails_helper"
+
+describe CloudWatchService do
+  describe "#week_submissions" do
+    let(:forms_env) { "test" }
+    let(:form_id) { 3 }
+    let(:datapoints) { [ { sum: } ] }
+    let(:sum) { 3.0 }
```

Comment:
Can we rename this as total submissions or something please?

Reply:
Yep, renamed to explain what that number actually is. It was called sum as that's what the response uses, but we can give it more detail :D